mtools/test/test_mplotqueries.py

```diff
@@ -0,0 +1,35 @@
+import sys
+from datetime import datetime, timedelta
+from random import randrange
+
+from dateutil import parser
+from nose.plugins.skip import SkipTest
+from nose.tools import raises
+
+import os
+
+import mtools
+from mtools.mplotqueries.mplotqueries import MPlotQueriesTool
+from mtools.util.logevent import LogEvent
+from mtools.util.logfile import LogFile
+
+
+class TestMPlotQueries(object):
+
+    def setup(self):
+        """Startup method to create mloginfo tool."""
```

Comment:
mloginfor? shouldn't it be mplogqueries ?

Reply:
Yes, duly noted in the list of changes to be made.